content/30.ecosystem/60.network-faucets.md

```diff
@@ -5,22 +5,14 @@ description:
 
 To access the testnet funds (Sepolia) you can use one of the following third party faucets:
 
-## %%zk_testnet_name%% faucet by Chainstack
-
-[Chainstack Faucet](https://faucet.chainstack.com/zksync-testnet-faucet) is an easy to use
-Multi-Chain Faucet. You can use Chainstack Faucet to claim %%zk_testnet_name%% **0.05 ETH
-every 24 hours**. Chainstack API key is required.
-
-## %%zk_testnet_name%% faucet by thirdweb
-
-Connect your wallet to the [thirdweb Faucet](https://thirdweb.com/zksync-sepolia-testnet) and enjoy the free ZKsync Sepolia!
-Supports over 350+ wallets, plus social logins.
-You can use the thirdweb faucet to claim %%zk_testnet_name%% **0.1 ETH every 24 hours**.
-
-## LearnWeb3
-
-You can use [LearnWeb3's %%zk_testnet_name%%](https://learnweb3.io/faucets/zksync_sepolia/)
-to claim %%zk_testnet_name%% **0.01 ETH per day**. GitHub authentication is required.
+## %%zk_testnet_name%% faucets
+
+| Faucet Provider                                  | Daily Amount of ETH | Requirements                     |
+|--------------------------------------------------|---------------------|----------------------------------|
+| [Chainstack](https://faucet.chainstack.com/zksync-testnet-faucet)      | 0.05 ETH            | Chainstack API key      |
+| [thirdweb](https://thirdweb.com/zksync-sepolia-testnet)        | 0.1 ETH             | Connect your wallet              |
+| [LearnWeb3](https://learnweb3.io/faucets/zksync_sepolia/)       | 0.01 ETH            | GitHub authentication   |
+| [Getblock](https://getblock.io/faucet/zksync-sepolia/)          | 0.01 ETH            | Getblock account        |
```

Comment:
can remove the GetBlock link below and looks like the B is capitalized. think the alchemy one under sepolia faucets can be moved to this table too

Reply:
Thanks for that, update the spelling and moved around the Alchemy faucet